pyproject.toml

```diff
@@ -18,12 +15,11 @@ authors = [
 ]
 maintainers = [
     {name = "Isaac Virshup", email = "ivirshup@gmail.com"},
-    {name = "Philipp Angerer", email = "philipp.angerer@helmholtz-muenchen.de"},
+    {name = "Philipp Angerer", email = "philipp.angerer@helmholtz-munich.de"},
     {name = "Alex Wolf", email = "f.alex.wolf@gmx.de"},
 ]
-readme = {file = "README.md", content-type="text/markdown"}
+readme = "README.md"
```

Comment:
I recall this throwing errors if I didn't specify the content-type. Was that flit specific?

Reply:
not for many years: https://github.com/pypa/flit/pull/169

Reply:
Pretty sure I was running into this more recently than that. But I believe `twine check` was the one complaining, so it's fine as long as the build check is passing.